### PR TITLE
Add MGN12 Euclid mount, based on Trident carriage

### PR DESCRIPTION
**Pull request by kageurufu**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

This adds a Trident/V2.4 MGN 12 carriage with an inset for mounting a [Euclid Probe](https://github.com/nionio6915/Euclid_Probe)

#### How was it tested? 

I am using it on my V2.4

#### Any background context you want to provide?

I was using the a omron-to-euclid bodge part for a while, and decided to make this to improve on it.

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ like so:
 | | [Side Panels](./JNP/Side-Panels) | Panels with intergrated hinge | :heavy_check_mark: |:x: |:x: |:x: |
 | joposter | [V0 Clippable LED Caselight](./joposter/v0_clippable_ledmounts) | LED Case light for the V0 that clips into the extrusions | :heavy_check_mark: | :x: | :x: | :x: | :x: |
 | JosAr | [Klicky Probe](./JosAr/Klicky-Probe) | Microswitch based inductive probe replacement | :x: | :heavy_check_mark: |:heavy_check_mark: | :x: |
+| KageUrufu | [Euclid Probe Carriage](./kageurufu/mgn12-euclid) | A MGN-12 carriage with an inset mount for a Euclid Probe | :x: | :heavy_check_mark: | :heavy_check_mark: | :grey_question: |
```
Comment (KiloQubit) [inline printer_mods/README.md:142]: Can you add a checkmark for the Trident column?  Last column, check the table header in the current commits in upstream.

File: printer_mods/kageurufu/mgn12-euclid/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Afterburner MGN12 Carriage with Integrated Euclid Probe
+=======================================================
+
+Euclid Probe by [nionio6915 (Github)](https://github.com/nionio6915/Euclid_Probe), https://www.euclidprobe.com/
+
+![Printed mount](./Images/real-bottomview.jpg)
+
+This is a modified version of the MGN12 carriage for a Voron Trident or modded Voron 2.4 with a cutout for mounting the upper Euclid PCB. 
+
+![CAD view of the Euclid Probe embedded in the carriage](Images/cad-bottomview.png)
```
Comment (KiloQubit) [inline printer_mods/kageurufu/mgn12-euclid/README.md:10]: Broken image, should be `Images/CAD-bottomview.png`

File: printer_mods/kageurufu/mgn12-euclid/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+This is a modified version of the MGN12 carriage for a Voron Trident or modded Voron 2.4 with a cutout for mounting the upper Euclid PCB. 
+
+![CAD view of the Euclid Probe embedded in the carriage](Images/cad-bottomview.png)
+
+Other than requiring a Euclid Probe, the only additional parts needed for the MGN12 carriage are two M3 heatset inserts for mounting the upper PCB. 
+
+For the Trident, the probe itself can be mounted [fixed onto the gantry](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Frame%20Mount)
+
+On a V2.4, you have the choice of [gantry mounting](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Frame%20Mount) or [fixed bed mounting](https://github.com/nionio6915/Euclid_Probe/tree/main/stls#:~:text=Voron%202.4%20Bed%20Rail%20Mount). The fixed mounting has the benefit of removing the chance of errantly picking up the probe after reaching a high enough Z.
```
Comment (KiloQubit) [inline printer_mods/kageurufu/mgn12-euclid/README.md:16]: all that text after `#` doesn't seem to actually scroll to the right STL, so maybe just specify the STL names here and link to the STLs directory.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Just a few README changes below.

**Comment by KiloQubit** [comment]
@kageurufu are you planning to update this PR to address the review?

**Comment by kageurufu** [comment]
@KiloQubit I've rebased this, and made the requested changes!

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the update!

Still broken image:
Screen Shot 2021-10-07 at 19 24 37

Broken link in the README, did you mean to swap the things in square brackets and parens?

    [https://www.euclidprobe.com/02-probe-docks-toolboard-mounts/](euclidprobe.com)

**Pull request merged**